service/telemetry/config_test.go

```diff
@@ -67,177 +63,178 @@ func TestLoadConfig(t *testing.T) {
 	}
 }
 
+// TODO: review where unmarshal tests should live
```

Comment:
I would just remove the code or keep it as is, leaving it commented does not seem useful to me (we always have git history). If we remove it, we can file an issue to track this.